crates/runc-shim/src/main.rs

```diff
@@ -43,8 +44,21 @@ fn parse_version() {
     }
 }
 
-#[tokio::main]
-async fn main() {
+fn main() {
+    let num_threads = std::env::var("RUSTMAXPROCS")
```

Comment:
Looks like rebase didn't go well. This change shouldn't be in this PR.

Reply:
I'v already updated.